task-07/README.md

```diff
@@ -20,14 +20,24 @@ ping6 <RIOT-IPv6-addr>%tapbr0
 Note: on MAC use `bridge0` instead of `tapbr0`.
 
 ## Task 7.2: Extend `gnrc_minimal` application
-* Add the `gnrc_udp` module to the application's [Makefile](Makefile)
+* Add the `gnrc_udp` module to the application's [Makefile](https://github.com/RIOT-OS/RIOT/blob/master/examples/gnrc_minimal/Makefile)
```

Comment:
@Quintol thanks for addressing the other comments. Would you please split this line, for example before the Makefile reference? We should be ready to go afterwards